data/transactions/logic/eval.go

```diff
@@ -158,6 +185,8 @@ type EvalParams struct {
 	// GroupIndex should point to Txn within TxnGroup
 	GroupIndex int
 
+	PastSideEffects []*EvalSideEffects
```

Comment:
please explain a reasoning behind "slide of pointers to EvalSideEffects" instead of "slice of EvalSideEffects"

Reply:
It was unintentional, I didn't realize slices could be modified without pointers. I'll change this.